lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -311,6 +315,22 @@ def get_progress_detail(self):
         progress = '%s/%s' % (str(json_data['current_score']), str(json_data['total_possible']))
         return progress
 
+    def assertExpectedLanguageInPreference(self, user, expected_language_code):
```

Comment:
@noraiz-anwar  Just a custom assert method like we have built in methods `self.assertEqual`. I have written a custom assert in this case `self. assertExpectedLanguageInPreference(*argrs)`